modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -4,11 +4,12 @@ include::_attributes.adoc[]
 endif::[]
 :page-aliases: scatter-gather-xml-reference.adoc
 
-The Scatter-Gather component is a routing event processor that processes a Mule event through different parallel processing routes that contain different event processors. Each route receives a reference to the Mule event and executes a sequence of one or more event processors. Each of these routes uses a separate thread to execute the event processors, and the resulting Mule event can be either the same Mule event without modifications or a new Mule event with its own payload, attributes, and variables. The Scatter-Gather component then combines the Mule events returned by each processing route into a new Mule event that is passed to the next event processor only after every route completes successfully.
+The Scatter-Gather component (`<scatter-gather />`) is a routing event processor that processes a Mule event through different parallel processing routes that contain different event processors, which are connector operations or other Mule components. Each route receives a reference to the Mule event and executes a sequence of one or more event processors. Each of these routes uses a separate thread to execute the event processors, and the resulting Mule event can be either the same Mule event without modifications or a new Mule event with its own payload, attributes, and variables. The Scatter-Gather component then combines the Mule events returned by each processing route into a new Mule event that is passed to the next event processor only after every route completes successfully.
 
-The Scatter-Gather component executes each route in parallel, not sequentially. Parallel execution of routes can greatly increase the efficiency of your Mule application and may provide more information than sequential processing.
+The Scatter-Gather component executes each route in parallel, not sequentially. Parallel execution of routes can greatly increase the efficiency of your Mule application and can provide more information than sequential processing.
 
-The Scatter-Gather component works with repeatable streams. It does not process nonrepeatable streams, which can be read only once before they are lost. By default, all streams are repeatable in Mule unless a component's streaming strategy is configured to be nonrepeatable.
+//TODO: clarify seemingly conflicting info: "It does not process nonrepeatable streams" vs. can be "configured to be nonrepeatable"
```

Comment:
Ivan: If payload is a stream, you need to read multiple times. If non-repeatable, can't do that. 

If prior to Scatter Gather, you set a component that returns a non-repeatable stream and Scatter Gather receives it, it won't work. 

non-repeatable stream: payload can be read only once via pointer and cannot be re-read
repeatable stream: Mule feature allowing for re-reading the stream

corner case for S-G to receive a non-repeatable stream

Reply:
ok. moving to new section